src/tests/ftest/pool/pool_svc.py

```diff
@@ -55,6 +56,42 @@ def check_leader(self, previous_leader=None, expect_change=True):
             self.log.info("Pool leader: current=%s", current_leader)
         return current_leader
 
+    def is_pool_rebuild_in_progress(self, max_retries=5, sleep_timer=5):
```

Comment:
Not sure what your intent here. Are you trying to wait until rebuild is done? If so, you can use `TestPool.wait_for_rebuild()`. or are you trying to do something while the state is "rebuild in progress"? If so, why are you not using the return value?

Reply:
I wasnt aware of the function wait_for_rebuild, the functions are quite similar. Changing code to use the function already coded.